apps/mobile/src/utils/system.ts

```diff
@@ -0,0 +1,25 @@
+import type { FC } from 'react';
+import { Keyboard, Platform } from 'react-native';
+import Clipboard from '@react-native-clipboard/clipboard';
+import { runtime } from '@walless/core';
+import { showNotificationModal } from 'modals/Notification';
+
+export const copy = async (value: string, prefix?: FC) => {
+	if (runtime.isMobile) {
+		Clipboard.setString(value);
+	} else {
+		await navigator.clipboard.writeText(value);
+	}
+
+	showNotificationModal({
+		id: 'copy',
+		prefix,
+		message: 'Copied',
+	});
```

Comment:
should not use showModal in utils, prefer separating it, the copy method just takes a copy, and showNotificationModal in which calling this method